docs/.gitignore

```diff
@@ -0,0 +1,7 @@
+* 
+!.gitignore 
+!conf.py
+!index.rst
+!_templates
+!_static 
+!_build
```

Comment:
Do the `_build`, `_static`, and `_templates` directories need to be checked into git, or will sphinx automatically create them on demand?

Reply:
they're necessary for sphinx, I'm pretty sure.